nemo_aligner/metrics/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# importing all available metrics here to make it easier to import from the config
+from nemo_aligner.metrics.common import ExactStringMatchMetric, InferenceMetricsHandler
```

Comment:
```suggestion
from nemo_aligner.metrics.common import InferenceMetricsHandler
```
missing import

Reply:
Sorry, was testing with that class in and forgot to remove the import after I removed it :)